store/interfaces.json

```diff
@@ -150,6 +150,12 @@
     "required": ["blog-search-list"],
     "allowed": ["blog-breadcrumb", "blog-search"]
   },
+  "store.not-found-product": {
```

Comment:
Why not?
```suggestion
  "store.not-found.product": {
```

Reply:
Waiting for info to see why this change do not with "admin/cms/pages"